EnableJupyterDisplay.swift

```diff
@@ -0,0 +1,178 @@
+#if canImport(Cryptor)
+import Cryptor
+#endif
+
+import Foundation
+
+enum JupyterDisplay {
+    struct Header: Codable {
+        let msgId: String
+        let username: String
+        let session: String
+        var date: String {
+            let currentDate = Date()
+            let formatter = DateFormatter()
+            formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"
+            formatter.timeZone = TimeZone(secondsFromGMT: 0)
+            formatter.locale = Locale(identifier: "en_US_POSIX")
+            return formatter.string(from: currentDate)
+        }
+        let msgType: String
```

Comment:
Avoid abbreviations.
```suggestion
        let messageType: String
```

Reply:
On doing this I'll need to add an enum with the CodingKeys. I can't add a key for date because it isn't a stored property. Therefore I have to make the Header conform to protocol Encodable, instead of Codable.